middleware/README.md

```diff
@@ -1010,7 +1010,7 @@ Indicates the type of SCU you'd want to reference to. Possible values for `type`
 * **cryptovision**
 * **dieboldnixdorf**
 * **epson**
-* **fiskalycertified**
+* **FiskalyCertified**
```

Comment:
Are you sure this is not supposed to be lower case?

Reply:
In all the Templates i created for the V2 it had to be upper case.

Reply:
All right, thanks.

@TSchmiedlechner, could you clarify whether fiskaly 2 requires camel case in templates? All other TSE types are referenced in lower case.

Reply:
![image](https://user-images.githubusercontent.com/47039335/135217951-1521ee8a-7082-4cbe-847a-5d5c6f5f6596.png)

Reply:
That's the package name, I was referring to the templating reference.

Reply:
As far as I read this from the code, both should work (i.e. the references are not case-sensitive).

Reply:
I'd keep it lower case then.

Reply:
@saschaknapp, could you please just change it back and I think we could merge it.

I already covered the translation at ec76410.